### PR TITLE
Google Ads Documentation Update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Standardized the doc and added updated connection settings and information on dynamic remarketing.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/advertising/google-ads.mdx [hunk 1]
```diff
@@ -5,67 +5,107 @@ description: Step-by-step guide to send event data from RudderStack to Google Ad
...
 
+- **Conversion Linker**: Enable this setting to use Google Ads' Conversion Linker feature.
+
+<div class="infoBlock">
+ 
+For more information on the Conversion Linker feature in Google Ads, refer to their official <a href="https://support.google.com/google-ads/answer/7521212?hl=en">documentation</a>.
 </div>
 
-- Give a name to the destination and click on **Next**. You should then see the following screen:
+- **Dynamic Remarketing**: Enable this setting to leverage the dynamic remarketing feature of Google Ads.
```
Comment (saikumarrs) [inline destinations/advertising/google-ads.mdx:68]: We need to also clarify that this is not a feature extension to the existing integration but a different tracking mode altogether.
@akashrpo

File: destinations/advertising/google-ads.mdx [hunk 2]
```diff
@@ -5,67 +5,107 @@ description: Step-by-step guide to send event data from RudderStack to Google Ad
...
+You can make a `page` call with the conversion name to RudderStack for a **Page Load Conversion** which will, in turn, send this data to Google Ads.
 
-A sample `page` call is as shown:
+A sample `page` call is as shown below:
 
 ```javascript
 rudderanalytics.page('page view');
 ```
 
+A sample `page` call after enabling the **Dynamic Remarketing** option in the RudderStack dashboard is shown below:
```
Comment (saikumarrs) [inline destinations/advertising/google-ads.mdx:92]: We should also clarify that the format for `parameters` is flexible. Either use the standard format Google expects for tracking specific categories or custom format for custom-defined categories.
@akashrpo to guide @ameypv-rudder on this.

File: destinations/advertising/google-ads.mdx [hunk 3]
```diff
@@ -5,67 +5,107 @@ description: Step-by-step guide to send event data from RudderStack to Google Ad
...
+});
+```
+
 ## Track
 
-You can make a track call with the conversion name to our RudderStack for a Click Event Conversion, which will in turn send the data to Google Ads.
+You can make a `track` call with the conversion name to RudderStack for a **Click Event Conversion** which will, in turn, send this data to Google Ads.
 
-RudderStack maps the `properties.value`, `properties.currency` or `properties.order_id` to the supported `value`, `currency`, and `transaction_id` fields of Google Ads, respectively. RudderStack also maps `properties.revenue` to `value`.
+RudderStack maps `properties.value`, `properties.currency`, and `properties.order_id` to the supported Google Ads `value`, `currency`, and `transaction_id` fields, respectively. RudderStack also maps `properties.revenue` to `value`.
```
Comment (saikumarrs) [inline destinations/advertising/google-ads.mdx:108]: This is strictly applicable when dynamic remarketing is turned off. We should word the document accordingly.

**Review by saikumarrs — COMMENTED** [review]
LGTM except for the suggestions I made.

File: destinations/advertising/google-ads.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply destinations/advertising/google-ads.mdx]: tweaked the instruction to clearly mention this.

File: destinations/advertising/google-ads.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply destinations/advertising/google-ads.mdx:103]: Do you think mentioning two snippets - one in each format will be helpful? @saikumarrs @akashrpo

File: destinations/advertising/google-ads.mdx (hunk 3 above)
Comment (ameypv-rudder) [reply destinations/advertising/google-ads.mdx]: Done!

File: destinations/advertising/google-ads.mdx (hunk 2 above)
Comment (saikumarrs) [reply destinations/advertising/google-ads.mdx:103]: @ameypv-rudder that'd be a good idea. @akashrpo thoughts?

Comment (akashrpo) [reply destinations/advertising/google-ads.mdx:103]: adding two snippets can provide better clarify on it.

File: destinations/advertising/google-ads.mdx (hunk 3 above)
Comment (saikumarrs) [inline destinations/advertising/google-ads.mdx]: Please convert my suggestion into a note.
```suggestion
RudderStack maps `properties.value`, `properties.currency`, and `properties.order_id` to the supported Google Ads `value`, `currency`, and `transaction_id` fields, respectively. RudderStack also maps `properties.revenue` to `value`.
Note that this is only applicable when Dynamic Re-marketing is turned off.
```

**Review by akashrpo — APPROVED** [review]
LGTM

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**